game-using-jsDOM/styles.css

```diff
@@ -0,0 +1,268 @@
+body {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    /* background-color: #eeeeee; */
+    background-color: #cfe0e8;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+.mainDiv {
+    display: flex;
+    flex-direction: column;
+    margin-top: 50px;
+    justify-content: center;
+    /* background-color: #c097cf; */
+    background-color: #eeeeee;
+    border-radius: 20px;
+    width: 1000px;
+    height: 800px;
+    
+}
+
+.titleOfGameDiv {
+    display:flex;
+    justify-content: center;
+    /* font-family: 'Elsie', cursive; */
+    font-family: cambria;
+    font-size: xx-large;
+}
+
+#word{
+    font-family: 'Great Vibes', cursive;
+    color: #8260a2;
+}
+
+.startButtonDiv {
+    display:flex;
+    justify-content: center;
+}
+
+.hideStartButton {
+    display: none;
+}
+
+.gameBoard {
+    display: flex;
+    justify-content: center;
+    /* background-color: red; */
+
+}
+.gameBoardDiv {
+    /* background-color: blue; */
+    width: 600px;
+    height: 400px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+.letterBoxesDiv {
+    /* background-color: teal;  */
+    justify-content: center;
+    display: flex;
+    padding: 20px;
+}
+
+.letterBox {
+    /* border-color: orange; */
+    margin: 5px 5px 5px 5px;
+    box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px, rgba(60, 64, 67, 0.15) 0px 1px 3px 1px;
+    padding: 20px;
+    height: 15px;
+    width: 10px;
+    /* border-bottom: 1px solid black; */
```

Comment:
I'm seeing several commented-out style properties across the file -- are these things that you can confidently remove?